update/Pos/Update/Mode.hs

```diff
@@ -7,18 +7,13 @@ module Pos.Update.Mode
 
 import           Universum
 
-import           Data.Tagged (Tagged)
+import           Control.Monad.Catch (MonadMask)
```

Comment:
Delete this import please. It's redundant and we tend to avoid `Control.Monad.Catch` imports in favor of `safe-exceptions` modules.

Reply:
Oh, does `MonadMask` come from `Universum`?

Reply:
Yes.